index.html

```diff
@@ -29,18 +29,19 @@
         </nav>
         </header>
         <section id="headline-section">
-                <h1 class="headline-title"> Hi there! <br>
```

Comment:
- [ ] Good job adding the desktop version to your portfolio 👏 , but I noticed that the hamburger button that you use in the mobile version is still present in your desktop version, kindly remove it 👍 

![Capture 5](https://user-images.githubusercontent.com/80458199/228805882-43da6d03-713b-495f-9831-51f15ca97805.PNG)